src/sagemaker/jumpstart/types.py

```diff
@@ -98,6 +97,25 @@ def __repr__(self) -> str:
         }
         return f"{type(self).__name__} at {hex(id(self))}: {str(att_dict)}"
 
+    def to_json(self) -> Dict[str, Any]:
```

Comment:
can you remove this implementation here: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/jumpstart/types.py#L898-L915